pyproject.toml

```diff
@@ -65,6 +65,7 @@ langchain = [
   "chromadb>=0.5.0",
   "pysqlite3",
   "opentelemetry-exporter-otlp",
+  "langchain-nvidia-ai-endpoints",
```

Comment:
This is not required for regular non-nvidia langchain users right?  Should this be in its own extra like `chat-nvidia`?

Reply:
Just to add to Andrew's comment. Currently you have put the integration under the integrations/langchain dir. Instead, you should create a new dir (eg `integrations/langchain_nvidia_ai_endpoints` then create a new list in here `langchain_nvidia_ai_endpoints=[...]` and list the requirements you need for these tests

Reply:
will do

Reply:
This is still under the `langchain` extra -- you need to make a new `chat-nvidia` extra